### PR TITLE
Add Actual.app Latest

### PR DESCRIPTION
**Pull request by denkristoffer**
After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by neutric** [comment]
@denkristoffer  Thank you for your contribution to `homebrew-cask`!
Please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

```
%  curl -I https://download.actualbudget.com/download/dmg                                                                                                                         
HTTP/2 302 
server: nginx/1.18.0
date: Sat, 06 Nov 2021 20:06:23 GMT
location: https://github.com/actualbudget/releases/releases/download/0.0.143/Actual-0.0.143.dmg
```
Also, since a versioned URL is available, please use it.

File: Casks/actual.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "actual" do
+  version :latest
+  sha256 :no_check
```
Comment (neutric) [inline Casks/actual.rb:3]: version "0.0.143"
sha256 "ab5baf1c99c0be468f2a7a4cab85108559f1d2ddf3073bb4966fb79a884d11b5"

File: Casks/actual.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  url "https://download.actualbudget.com/download/dmg"
```
Comment (neutric) [inline Casks/actual.rb:5]: url "https://github.com/actualbudget/releases/releases/download/#{version}/Actual-#{version}.dmg"

File: Casks/actual.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name "Actual"
+  desc "Super fast privacy-focused app for managing your finances"
```
Comment (neutric) [inline Casks/actual.rb:7]: Please read the [dos and don'ts](https://docs.brew.sh/Cask-Cookbook#stanza-desc).

**Review by neutric — COMMENTED** [review]
I added a few comments directly in the code.

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @denkristoffer.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

**Comment by neutric** [comment]
```
zap trash: [
  "~/Library/Application Support/Actual",
  "~/Library/Logs/Actual",
  "~/Library/Preferences/com.shiftreset.actual.plist",
  "~/Library/Saved Application State/com.shiftreset.actual.savedState",
]
```

**Comment by miccal** [comment]
Thank you @denkristoffer.

**Pull request merged**